addon/services/user-activity.js

```diff
@@ -45,6 +44,7 @@ export default Service.extend(Evented, FastBootCompatMixin, {
 
   init() {
     this._super(...arguments);
+    this.set('defaultEvents', ['keydown', 'mousedown', 'scroll', 'touchstart']);
```

Comment:
What prompted this change? I know that this is generally discourages because it shares across instances, but services are singletons. This array can actually be overwritten by the consuming app, so I'm worried that moving this to init would cause the addon to overwrite the override.

Reply:
Alternatively, if you check for an existing array and only write if it doesn't exist, that would be fine.

Reply:
Ok, so I went back and forth on this.  The blueprint for 2.18 includes `eslint-ember-plugin` which has a rule to avoid leaky state.  I decided to follow the rule instead of a `eslint-dsiable` on these files.  I prefer reverting because following the rule caused me to adjust the tests so that they would start passing again.  What would like me to do?